Maple2.Server.Game/Model/Field/Actor/FieldNpc.cs

```diff
@@ -89,8 +90,8 @@ public class FieldNpc : Actor<Npc> {
     private MS2PatrolData? patrolData;
     private int currentWaypointIndex = 0;
 
-    public FieldNpc(FieldManager field, int objectId, Agent? agent, Npc npc, string? patrolDataUUID = null) : base(field, objectId, npc) {
-        IdleSequence = npc.Animations.GetValueOrDefault("Idle_A") ?? new AnimationSequence(-1, 1f, null);
+    public FieldNpc(FieldManager field, int objectId, Agent? agent, Npc npc, string? patrolDataUUID = null) : base(field, objectId, npc, npc.Metadata.Model, field.NpcMetadata) {
+        IdleSequence = npc.Animations.GetValueOrDefault("Idle_A") ?? new AnimationSequence("Idle_A", -1, 1f, null);
```

Comment:
I dont think this makes sense, if we can't get animation Idle_A, we are still setting the animation as Idle_A?